step.yml

```diff
@@ -1,44 +1,53 @@
 title: Xcode Archive & Export for iOS
-summary: Run the Xcode archive command and then export an .ipa from the archive.
+summary: Automatically manages your code signing assets, archives and exports an .ipa in one Step.
 description: |-
-
-  The Step archives your Xcode project by running the `xcodebuild archive` command and then exports the archive into an .ipa file with the `xcodebuild -exportArchive` command. This .ipa file can be shared, installed on test devices, or uploaded to the App Store Connect.
+  With this Step, you can use automatic code signing in a [CI environment without having to use Xcode](https://developer.apple.com/documentation/xcode-release-notes/xcode-13-release-notes).
+  In short, the Step:
+  - Logs you into your Apple Developer account based on the [Apple service connection you provide on Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
```

Comment:
```suggestion
  - Downloads certificates and profiles from your Apple Developer account based on the [Apple service connection you provide on Bitrise](https://devcenter.bitrise.io/en/accounts/connecting-to-services/apple-services-connection.html).
```

Reply:
Oh I see the next bullet point below, nevermind (or feel free to merge them if it's better)

Reply:
@ofalvai I'm happy to add certificates to the sentence but are we positive the Step takes care of that too? I'm confident it does with the prov profiles but unsure about certs 🤔  😅

Reply:
Good point, it only downloads profiles 😱

Reply:
cool! :)